.github/CODEOWNERS

```diff
@@ -11,7 +11,7 @@ yarn.lock                                           @backstage/community-plugins
 /workspaces/3scale                                  @backstage/community-plugins-maintainers  @04kash @AndrienkoAleksandr
 /workspaces/acr                                     @backstage/community-plugins-maintainers  @christoph-jerolimov @invincibleJai
 /workspaces/adr                                     @backstage/community-plugins-maintainers  @kuangp
-/workspaces/analytics                               @backstage/community-plugins-maintainers  @jmezach
+/workspaces/analytics                               @backstage/community-plugins-maintainers  @jmezach @schultzp2020 @christoph-jerolimov
```

Comment:
Thanks for adding @schultzp2020 and me, but are we hi-jacking another workspace here?

@jmezach, is it fine for you if we just take care of the extra modules here, or are you fine if we also approve code changes to your modules?

Or, @awanlin @BethGriggs @04kash, should we consider adding additional lines here just for the module(s) we added? It's this PR for Google Analytics and #1906 for Matomo.

I just want to be a kindful and transparent citizen of this repo. :peace_symbol: :v:

Reply:
I've added this as a topic for the next Community SIG, but my thinking is to have paths to each plugin in the `CODEOWNERS` file. @jmezach contributed the New Relic analytics plugin, the GA ones are from Spotify from what I can remember.